src/lib/cloud/digitalocean/provisioner.ts

```diff
@@ -33,13 +31,89 @@ const REDIRECT_URIS: [string] = [
   //  'http://localhost:10101'
 ];
 
+const STORAGE_KEY_OAUTH = 'DigitalOcean-OAuth';
+
 interface KeyPair {
   private: string;
   public: string;
 }
 
 class Provisioner {
-  constructor(private dispatch_: Function, private state_: any = {}) {}
+  constructor(
+    private sshKeyPair_ :any = {},
```

Comment:
Amended in d888653 - added a comment about that implicit `dispatch_` field. I take it it's not concerning that everything seemed to work just fine without it (other than from an "explicit is better than implicit" perspective (and probably other [zen](https://www.python.org/dev/peps/pep-0020/#id3) insults))?